app/modules/listings/components/listing_element.svelte

```diff
@@ -20,75 +24,70 @@
     imageUrl = image.url
   }
 </script>
-
-<li>
-  <a
-    href="/entity/{uri}"
-    title={label}
-    on:click={loadInternalLink}
-  >
-    {#if imageUrl}
-      <ImagesCollage
-        imagesUrls={[ imageUrl ]}
-        imageSize={100}
-        limit={1}
+<div class="listing-element-section">
+  <div class="listing-element">
+    <a
+      href="/entity/{uri}"
+      title={label}
+      on:click={loadInternalLink}
+    >
+      {#if imageUrl}
+        <ImagesCollage
+          imagesUrls={[ imageUrl ]}
+          imageSize={100}
+          limit={1}
+        />
+      {/if}
+      <div>
+        <span class="label">{label}</span>
+        {#if publicationYear}
+          <p
+            class="publicationYear"
```

Comment:
```suggestion
            class="publication-year"
```
Please use dashed case for CSS classes with no JS behavior (so that we visually know when a class has some weird JS behavior that needs to be kept in mind)

Reply:
fixed in b48d43ab6